biodatasets/meqsum/meqsum.py

```diff
@@ -0,0 +1,147 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Dataset for medical question summarization introduced in the ACL 2019 paper "On the Summarization of Consumer Health
+Questions". Question understanding is one of the main challenges in question answering. In real world applications,
+users often submit natural language questions that are longer than needed and include peripheral information that
+increases the complexity of the question, leading to substantially more false positives in answer retrieval. In this
+paper, we study neural abstractive models for medical question summarization. We introduce the MeQSum corpus of 1,000
+summarized consumer health questions.
+"""
+
+import os
+from typing import Dict, List, Tuple
+
+import datasets
+import pandas as pd
+
+from utils import schemas
+from utils.configs import BigBioConfig
+from utils.constants import Tasks
+
+_CITATION = """\
+@Inproceedings{,
+    author = {Asma {Ben Abacha} and Dina Demner-Fushman},
+    title = {On the Summarization of Consumer Health Questions},
+    booktitle = {Proceedings of the 57th Annual Meeting of the Association for Computational Linguistics,
+                 ACL 2019, Florence, Italy, July 28th - August 2},
+    year = {2019},
+}
+"""
+
+_DATASETNAME = "meqsum"
+
+_DESCRIPTION = """\
+Dataset for medical question summarization introduced in the ACL 2019 paper "On the Summarization of Consumer Health
+Questions". Question understanding is one of the main challenges in question answering. In real world applications,
+users often submit natural language questions that are longer than needed and include peripheral information that
+increases the complexity of the question, leading to substantially more false positives in answer retrieval. In this
+paper, we study neural abstractive models for medical question summarization. We introduce the MeQSum corpus of 1,000
+summarized consumer health questions.
+"""
+
+_HOMEPAGE = "https://github.com/abachaa/MeQSum"
+
+_LICENSE = "Unknown"
+
+_URLS = {
+    _DATASETNAME: "https://github.com/abachaa/MeQSum/raw/master/MeQSum_ACL2019_BenAbacha_Demner-Fushman.xlsx",
+}
+
+_SUPPORTED_TASKS = [Tasks.SUMMARIZATION]
+
+_SOURCE_VERSION = "1.0.0"
+
+_BIGBIO_VERSION = "1.0.0"
+
+
+class MeQSumDataset(datasets.GeneratorBasedBuilder):
+    """TODO: Short description of my dataset."""
```

Comment:
Could you only update the description with something like:

```suggestion
    """Dataset containing 1000 summarized consumer health questions."""
```